openmetadata-ui/src/main/resources/ui/src/components/DatasetDetails/DatasetDetails.interface.ts

```diff
@@ -26,19 +27,17 @@ export interface DatasetDetailsProps {
   tableDetails: Table;
   datasetFQN: string;
   dataModel?: Table['dataModel'];
-  activeTab: number;
+  activeTab: EntityTabs;
   tableProfile: Table['profile'];
-  sampleData: TableData;
   entityThread: Thread[];
   isTableProfileLoading?: boolean;
-  isSampleDataLoading?: boolean;
   isEntityThreadLoading: boolean;
   feedCount: number;
   entityFieldThreadCount: EntityFieldThreadCount[];
   entityFieldTaskCount: EntityFieldThreadCount[];
   paging: Paging;
   createThread: (data: CreateThread) => void;
-  setActiveTabHandler: (value: number) => void;
+  setActiveTabHandler: (value: string) => void;
```

Comment:
Do we need this handler to handle tabs on the details component?
AntD will handle the tab changes and if we want to get the active tab we can get it through `useParams`.

Reply:
Yes, I was also thinking on that way, let me try it out, thanks for pointing it out!